File: app/src/main/java/de/jbamberger/irremote/ui/TestActivity.java
package de.jbamberger.irremote.ui;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.support.v7.widget.LinearLayoutManager;
import android.support.v7.widget.RecyclerView;
import android.support.v7.widget.Toolbar;

import de.jbamberger.irremote.R;

public class TestActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_test);
        Toolbar toolbar = (Toolbar) findViewById(R.id.toolbar);
        setSupportActionBar(toolbar);

        RecyclerView recyclerView = (RecyclerView) findViewById(R.id.content_empty);
        recyclerView.setHasFixedSize(true);
        recyclerView.setLayoutManager(new LinearLayoutManager(this));

        BaseAdapter adapter = new BaseAdapter(this);
        recyclerView.setAdapter(adapter);
    }
}

File: app/src/main/java/de/jbamberger/irremote/ui/TestActivity.kt
package de.jbamberger.irremote.ui

import android.os.Bundle
import android.widget.Button
import androidx.appcompat.app.AppCompatActivity
import de.jbamberger.irremote.R

class TestActivity : AppCompatActivity() {

    lateinit var remoteBase: RemoteBaseView

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_test)

        remoteBase = findViewById(R.id.remoteBase)

        val addButtonButton = findViewById<Button>(R.id.addButtonButton)
        addButtonButton.setOnClickListener {
            remoteBase.addBtn(100f, 100f)
        }
    }
}
